contrib/bash-completion

```diff
@@ -23,9 +23,17 @@ _netctl() {
         COMPREPLY=( $(compgen -W "--help --version list store restore stop-all start stop restart switch-to is-active status enable disable reenable is-enabled edit wait-online" -- "$cur") )
       ;;
       2)
-        [[ ${COMP_WORDS[COMP_CWORD-1]} = @(start|stop|restart|switch-to|is-active|status|enable|disable|reenable|is-enabled|edit|wait-online) ]] &&
+        [[ ${COMP_WORDS[COMP_CWORD-1]} = @(is-active|status|enable|disable|reenable|is-enabled|edit|wait-online) ]] &&
```

Comment:
This command has a non-zero return value if the argument to `netctl` is not recognized. I think this is conceptually fine. By splitting some arguments, it has a non-zero return value also in acceptable situations. Let's turn this into an `if ... elif` block.

Reply:
Isn't current commit the same thing as `if ... elif` block? The union of all the values in 3 `[[ ${COMP_WORDS[ ...` branches add up to a set of values in `COMPREPLY`.

Reply:
Imagine what things would look like if we had `set -e`, somewhere. Suddenly it makes a big difference whether individual statements return a non-zero value.

Reply:
I didn't know about `set -e`, but `man set` says this:
```
APPLICATION USAGE
       Application writers should avoid relying on set −e within functions.
```
Can you make a code suggestion for `if ... elif` code block please? I'm not sure what you are aiming at here.

Reply:
something like this (untested):
```
        if [[ ${COMP_WORDS[COMP_CWORD-1]} = @(is-active|status|enable|disable|reenable|is-enabled|edit|wait-online) ]]; then
            compopt -o filenames
            mapfile -t COMPREPLY < <(IFS=$'\n'; compgen -W "$(_netctl_profiles)" -- "$cur")
        elif [[ ${COMP_WORDS[COMP_CWORD-1]} = @(stop|restart) ]]; then
            # Suggest only active profiles for stopping/restarting
            compopt -o filenames
            mapfile -t COMPREPLY < <(IFS=$'\n'; compgen -W "$(netctl list | sed -n 's/^[^ ] //p')" -- "$cur")
        elif [[ ${COMP_WORDS[COMP_CWORD-1]} = @(switch-to|start) ]]; then
            # Suggest only inactive profiles for starting/switching-to
            compopt -o filenames;
            mapfile -t COMPREPLY < <(IFS=$'\n'; compgen -W "$(netctl list | sed -n 's/^  //p')" -- "$cur")
        fi
```